### PR TITLE
[DataGrid] Add Italian (itIT) locale

### PR DESCRIPTION
**Pull request by profcav**

### CODE REVIEW

File: docs/src/pages/components/data-grid/localization/localization.md [hunk 1]
```diff
@@ -60,6 +60,7 @@ The same import works with `XGrid` as it's an extension of `DataGrid`.
 | Portuguese (Brazil)     | pt-BR               | `ptBR`      |
 | Spanish (Spain)         | es-ES               | `esES`      |
 | Ukraine                 | uk-UA               | `ukUA`      |
+| Italian                 | it-IT               | `itIT`      |
```
Comment (oliviertassinari) [inline docs/src/pages/components/data-grid/localization/localization.md:63]: Please keep the table sorted ASC, same for the imports

File: packages/grid/_modules_/grid/locales/itIT.ts [hunk 2]
```diff
@@ -0,0 +1,87 @@
+import { itIT as itITCore } from '@material-ui/core/locale';
+import { GridLocaleText } from '../models/api/gridLocaleTextApi';
+import { getGridLocalization, Localization } from '../utils';
+
+const itITGrid: Partial<GridLocaleText> = {
+  // Root
+  rootGridLabel: 'grid',
```
Comment (oliviertassinari) [inline packages/grid/_modules_/grid/locales/itIT.ts:7]: https://github.com/mui-org/material-ui-x/issues/1290

```suggestion
```

**Review by oliviertassinari — DISMISSED** [review]

File: docs/src/pages/components/data-grid/localization/localization.md (hunk 1 above)
Comment (profcav) [reply docs/src/pages/components/data-grid/localization/localization.md:63]: ok, moving Dutch line too (already not sorted)

**Review by DanailH — APPROVED** [review]

**Comment by oliviertassinari** [comment]
@DanailH Feel free to merge

**Pull request merged**